astropy/io/fits/tests/test_hdulist.py

```diff
@@ -178,6 +178,16 @@ def test_append_groupshdu_to_non_empty_list(self):
         with pytest.raises(ValueError):
             hdul.append(hdu)
 
+    def test_append_scaled_image_with_do_not_scale_image_data(self):
+        """Tests appending a scaled ImageHDU to a HDUList."""
+
+        with fits.open(self.data("scale.fits"), do_not_scale_image_data=True) as source:
```

Comment:
I think we can chain these two `with` with a comma and save a level of indentation? 🤔